BondageClub/Scripts/Typedef.d.ts

```diff
@@ -123,6 +123,73 @@ declare function LoseFocus(event: MouseEvent): void;
 
 //#endregion
 
+//#region Chat
+
+interface ChatRoom {
+	Name: string;
+	Description: string;
+	Admin: number[];
+	Ban: number[];
+	Limit: number;
+	Game: any;
+	Background: string;
+	Private: boolean;
+	Locked: boolean;
+	BlockCategory: "Photos";
+	Character?: Character[];
```

Comment:
The type here is actually character data from server, not created character. I don't think we have types for that
```suggestion
	Character?: any[];
```